app.js

```diff
@@ -0,0 +1,38 @@
+const express = require('express');
```

Comment:
One of the things I've learned is that it's generally preferred to use `const` instead of `var` when setting variables like this. It's interesting that when running express-generate it uses var.